libs/agent-sm/build.gradle

```diff
@@ -20,3 +20,7 @@ base {
 
 test.enabled = false
 testingConventions.enabled = false
+
+dependencies {
+  compileOnly "com.github.spotbugs:spotbugs-annotations:4.9.7"
```

Comment:
Why was this added?

Reply:
See https://github.com/opensearch-project/OpenSearch/pull/19683#issuecomment-3418860536